src/makeUrls.ts

```diff
@@ -4,6 +4,7 @@ export interface CalendarEvent {
   location: string | null;
   startsAt: string;
   endsAt: string;
+  addresses?: string[]
```

Comment:
This file and changes are the main bit that I wanted to share, sorry this is a bad PR. You can add recipients to google cal invites